src/histograms/utils/HistogramUtils.ts

```diff
@@ -419,3 +425,80 @@ export const tickNumberFormat = (d, formatChar) => {
   const y = +format('')(d);
   return formatNumber(y, formatChar);
 };
+
+export const SELECTED_STYLE: Style = {
+  fill: '#5CCBC3',
+  stroke: '#5CCBC3',
+  stroke_width: 1,
+  background_color: '#FFF',
+  background_opacity: 0.2
+};
+
+export const UNSELECTED_STYLE: Style = {
+  fill: '#DADADA',
+  stroke: '#DADADA',
+  stroke_width: 1,
+  background_color: '#FFF',
+  background_opacity: 0
+};
+
+export const HEAD_BAR = {
+  SELECTED_STYLE,
+  UNSELECTED_STYLE,
+  HEIGHT: 5
+};
+
+export const BAR_OPTIONS = {
+  WEIGHT: 0.8,
+  BACKGROUND_COLOR: '#FFF'
+};
+
+export function getBarOptions(barOptions: BarOptions): BarOptions {
+  const returnedBarOptions: BarOptions = barOptions ? Object.assign({}, barOptions) : {};
+  if (returnedBarOptions.bar_weight === undefined) {
+    returnedBarOptions.bar_weight = BAR_OPTIONS.WEIGHT;
+  }
+  if (!returnedBarOptions.head_band) {
```

Comment:
You may also define default values with Object.assign({}, defaultValues, returnedBarOptions) with defaultValues a single object with all default values.  It may make it more readable as all default values are at the same place and there are no repeated `if`. Maybe it doesn't work in this case. Do as you want...